rgw/v2/tests/s3_swift/reusables/bucket_notification.py

```diff
@@ -105,17 +105,32 @@ def put_bucket_notification(
         raise TestExecError("put bucket notification failed")
 
 
-def get_bucket_notification(rgw_s3_client, bucketname):
+def put_empty_bucket_notification(rgw_s3_client, bucketname):
+    """
+    put empty bucket notification. Ref BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2017389
+    """
+    log.info(f"put empty bucket notification on {bucketname}")
+    put_empty_bkt_notification = rgw_s3_client.put_bucket_notification_configuration(
+        Bucket=bucketname,
+        NotificationConfiguration={},
+    )
+    if put_empty_bkt_notification is False:
+        raise TestExecError("put empty bucket notifcation failed")
+    get_bucket_notification(rgw_s3_client, bucketname, empty=True)
+
+
+def get_bucket_notification(rgw_s3_client, bucketname, empty=True):
```

Comment:
Could you please change the value of `empty` variable here

Reply:
@udaysk23 ,changed the flag 'empty' to false by default.